pavelib/assets.py

```diff
@@ -239,13 +430,18 @@ def compile_sass(options):
         source_comments = False
         output_style = 'compressed'
 
-    timing_info = []
-    system_sass_directories = applicable_sass_directories(systems)
-    all_sass_directories = applicable_sass_directories()
-    dry_run = tasks.environment.dry_run
-    for sass_dir in system_sass_directories:
+    for dirs in sass_dirs:
         start = datetime.now()
-        css_dir = sass_dir.parent / "css"
+        css_dir = dirs['css_destination_dir']
+        system_sass_dir = dirs['sass_source_dir']
+        lookup_paths = dirs['lookup_paths']
+
+        if not system_sass_dir.isdir():
+            print("\033[91m Sass dir '{dir}' does not exists, skipping sass compilation for '{theme}' \033[00m".format(
+                dir=sass_dirs, theme=theme or system,
+            ))
+            # invalid sass directory, skip sass compilation
```

Comment:
Isn't it valid for a theme to not provide a particular sass directory? I think the comment shouldn't sound like this was an error, but should be something like:

    # theme doesn't override sass directory, so skip it